scripts/lsstinstall

```diff
@@ -348,6 +348,14 @@ else
     echo "$EUPS_PKGROOT" > "$EUPS_PATH/pkgroot" || fail "Unable to write pkgroot file"
 fi
 
+# Copy postinstall
+callback_folder="$EUPS_DIR/site"
```

Comment:
Check that `lsstinstall` knows the value of `$EUPS_DIR` and isn't inheriting it from the shell.

Reply:
It looks like `$EUPS_PATH/site` does actually work.